INSTALL-Linux.md

```diff
@@ -0,0 +1,128 @@
+
+These examples assume you have the Mixxx source code at `~/mixxx`. If you have it elsewhere, use that path instead in the following commands.
+
+```shell
+cmake -DCMAKE_INSTALL_PREFIX=/usr/local -S ~/mixxx -B ~/mixxx/build
+```
+
+# Compile
+```shell
+cmake --build ~/mixxx/build --parallel `nproc`
+```
+
+# Install
+```shell
+cmake --build ~/mixxx/build --target install --parallel `nproc`
+```
+
+If you want to compile and install in one step, you can skip the compilation step above and just run this command.
+
+# Run without installing
+The `mixxx` binary will be in the CMake build directory. You can simply run it directly:
+```shell
+~/mixxx/build/mixxx
+```
+
+# ccache
+
+We highly recommend installing [CCache](https://ccache.dev/) if you are building Mixxx frequently, whether for development or testing. CCache drastically speeds up the time to recompile Mixxx, especially when switching between git branches. CMake works with CCache automatically.
+
+You will probably want to increase the default ccache size of 5.0GB to something much larger to accommodate Mixxx's large build sizes. You can adjust the cache size with the --set-config flag:
+```sh
+ccache --set-config=max_size=20.0G
+```
+
+# Development tips
+
+## Debug build and assertions
+
+If you want to make a debug build, add `-DCMAKE_BUILD_TYPE=Debug -DDEBUG_ASSERTIONS_FATAL=ON` to the end of the cmake configure command.
+Debug builds should be started with the command line option `--debugAssertBreak` to trigger a breakpoint in the debugger if debug
+assertions are violated or to abort Mixxx immediately. This ensures that messages about violated debug assertions are not missed between various other debug log messages. We recommend this if you are working on Mixxx code, but not if you are performing with Mixxx.
+
+## Non-System Qt
+
+To build Mixxx with a version of Qt older or newer than your distribution's package manager, download the latest [Qt source
+code](https://download.qt.io/archive/qt/). For each Qt version, it is available at that link in a directory called "single" and has a filename like `qt-everywhere-src-VERSION.tar.xz`. Extract that archive and compile the source code:
+
+```shell
+tar xf qt-everywhere-src-VERSION.tar.xz
+cd qt-everywhere-src-VERSION
+./configure -prefix /path/to/qt/install -system-sqlite -sql-sqlite -qt-zlib -opensource -confirm-license -nomake examples -nomake tests -skip qt3d -skip qtwebengine
+make -j`nproc`
+make install
+```
+
+Append `-DCMAKE_PREFIX_PATH=/path/to/qt/install` (where `/path/to/qt/install` is the path you used when building Qt) to the cmake configure command to instruct cmake to prefer the Qt version from that path.
```

Comment:
This should be removed. We can just put a link to Qt's build instructions if it's really deemed necessary to mention this explicitly. But IMHO the better approach is to just explain what CMAKE_PREFIX_PATH can be used to add additional search directories for dependencies.

Reply:
"... for example to build with non-system Qt (link to our wiki in case our instructions are easier to digest than having to collect it from various stackoverflow answers)